### DISCUSSION TITLE
What is this path: /__nextjs_original-stack-frame?

### DISCUSSION THREAD

**Discussion by hmessing** [vercel/next.js#63092] [App Router]
I'm using Clerk to do very simple authentication with much of the site not password protected, and its middleware occasionally gives me the error that "The request to /__nextjs_original-stack-frame is being redirected " as it is not in the list of public or ignored routes. It suggests I add it to one of those. I am hesitant to do that without knowing what it is. Any help, please?

**Comment by emil14** [comment]
any updates?

**Comment by hmessing** [comment]
Nope

**Comment by thegdsks** [comment]
No Updates yet ?

**Comment by sathishkannan162** [comment]
any updates?

**Comment by hmessing** [comment]
Sorry no nothing

**Comment by thekbbohara** [comment]
any updates 😭😭

**Comment by hmessing** [comment]
nope

**Comment by Netail** [comment]
It's a call by NextJS to fetch the stack trace of an error, which is then displayed on the dev/error overlay

**Comment by coltonehrman** [comment]
As @Netail mentioned, this seems to be the related code: https://github.com/vercel/next.js/blob/0b699b1c8d6cca8e9ab39b4bc38e67945373f918/packages/next/src/client/components/react-dev-overlay/utils/stack-frame.ts#L95

**Reply by berkaycirak** [reply]
Any updates? it arises on dev environment and annoying

**Comment by aporroluyo3** [comment]
any updates yet?

**Comment by aliaz0** [comment]
The issue was resolved for me by disabling a browser extension.

**Reply by Peeja** [reply]
@aliaz0 What extension was it?

**Comment by berkaycirak** [comment]
Just update next version to `15.3.0` guys

**Comment by hugocruzlfc** [comment]
I had the same problem using Authjs (formerly Next-Auth) in a version of Nextjs 15.1.4. I solved it by updating all dependencies.

**Comment by blooser** [comment]
Hey!

Any updates? It is super annoying, the requests are super slow, it's slowing my frontend and sometimes it may spam a lot of (it've got even 400 that requests in just one minute! )

![image](https://github.com/user-attachments/assets/f7929597-a668-4d58-9450-2bb4c97a6d7b)

**Reply by vucinatim** [reply]
i am having the same issue. 
    "next": "15.0.4",
    "payload": "3.28.1",

**Reply by Ahmad-code077** [reply]
@hugocruzlfc  did the issue resolve by updating all dependencies or i saw this happen only in the development not in the production 
I was checking it for one day but still the issue is same 
<img width="640" alt="Screenshot 2025-05-14 020942" src="https://github.com/user-attachments/assets/141b767c-490b-40c2-8f1f-f86b3a097f3b" />

**Reply by Ahmad-code077** [reply]
> @hugocruzlfc did the issue resolve by updating all dependencies or i saw this happen only in the development not in the production I was checking it for one day but still the issue is same <img alt="Screenshot 2025-05-14 020942" width="640" src="https://private-user-images.githubusercontent.com/146346428/443399502-141b767c-490b-40c2-8f1f-f86b3a097f3b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDcxNzA5MDksIm5iZiI6MTc0NzE3MDYwOSwicGF0aCI6Ii8xNDYzNDY0MjgvNDQzMzk5NTAyLTE0MWI3NjdjLTQ5MGItNDBjMi04ZjFmLWY4NmIzYTA5N2YzYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNTEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDUxM1QyMTEwMDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01YTgyMmRmZDNkNmQwMGExOWJjM2M2NzMxOGEwNDI1YmUzZTNhMzEyOGMyODdjZDFkNjU1MGFlN2UxMmMzYzg5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.jayzoI9Eh66z8RMgKOxB57kVO6epIec1lyX2rf5mZ9o">


Hey My issue is resolve by removing dependencies and reinstall them also by running 
npm audit fix i hope this help someone

**Comment by oleksii-kotvytskyi** [comment]
The response from [Netail](https://github.com/Netail) looks right. 
I have fixed the issues were in the application and after it, requests were not sent anymore. 

p.s. error stack trase you can find in the console

**Comment by camjackson-bigwx** [comment]
I was having problems with this, it turned out to be that Sentry was constantly trying to load this URL whenever an error occurred. Upgrading to the latest sentry SDK resolved it for me.
